tests/pytests/test.py

```diff
@@ -2152,6 +2152,7 @@ def testTimeoutOnSorter(env):
 
     res = env.cmd('ft.search', 'idx', '*', 'SORTBY', 'n', 'DESC')
     env.assertGreater(elements, res[0])
+    env.assertGreater(len(res), 2)
```

Comment:
Is that enough?
`env.assertGreater(len(res), 2)`
Should we also test the content of the result?

Reply:
No, because the content might be different each time..

Reply:
Its a timeout test...